.github/workflows/check.yml

```
@@ -36,6 +37,15 @@ jobs:
     needs: path-filter
     uses: ./.github/workflows/_test.yml
     permissions: {}
+    with:
+      node: [ "20", "lts/*" ]
```

actionlint (1.7.4)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)